### PR TITLE
Improve events

### PR DESCRIPTION
**Pull request by fnurkla**
This fixes a couple of the bugs listed in #188 but also makes dates not in $CURRENT_YEAR say what year they are in.

### CODE REVIEW

**Comment by danieladugyan** [comment]
I checked the bottom three issues off the list based on this MR, let me know if that's wrong or if I missed anything.

image

File: src/lib/components/TagSelector.svelte [hunk 1]
```diff
@@ -3,7 +3,9 @@
   import TagChip from "./TagChip.svelte";
 
   /** Called when the list of selected tags changes */
-  export let onChange: () => void = () => undefined;
+  export let onChange: () => void = () => {
+    searchValue = "";
+  };
```
[the remark this answers is not in the document]
Comment (fnurkla) [reply src/lib/components/TagSelector.svelte:8]: This is fixed with @alfredgrip's recent commits

**Comment by alfredgrip** [comment]
The current CI error is due to incompatible types. In the schema defined for creating events, `shortDescription` is nullable, but this is declared to be NOT nullable in Prisma. I agree that it shouldn't be nullable.

**Comment by danieladugyan** [comment]
> The current CI error is due to incompatible types. In the schema defined for creating events, `shortDescription` is nullable, but this is declared to be NOT nullable in Prisma. I agree that it shouldn't be nullable.

I don't know about that. AFAIK we don't even use them today (they're called short description on the React page) and I find it a bit annoying to have to come up with them.

**Comment by alfredgrip** [comment]
> I don't know about that. AFAIK we don't even use them today (they're called short description on the React page) and I find it a bit annoying to have to come up with them.

Fair enough, my point was that if are to keep that field, it should be non-nullable IMO. But we might as well remove it if it's not used

**Review by danieladugyan — APPROVED** [review]

**Pull request merged**